src/main/kotlin/no/nav/bidrag/bidragskalkulator/controller/PersonController.kt

```diff
@@ -53,12 +59,32 @@ class PersonController(
         return runBlocking(Dispatchers.IO + MDCContext()) {
             secureLogger.info { "Henter informasjon om pålogget person $personIdent og personens barn" }
 
-           brukerinformasjonService.hentBrukerinformasjon(personIdent).also {
+            brukerinformasjonService.hentBrukerinformasjon(personIdent).also {
                 secureLogger.info { "Henter informasjon om pålogget person $personIdent fullført" }
             }
 
         }
-
     }
 
-}
+    @Operation(
+        summary = "Henter grunnlagsdata for kalkulering uten autentisering",
+        description = "Henter underholdskostnader og samværsfradrag som brukes i bidragskalkuleringen. Returnerer 200 ved vellykket henting, eller passende feilkoder."
+    )
+    @ApiResponses(
+        value = [
+            ApiResponse(responseCode = "200", description = "Kalkuleringsinformasjon hentet vellykket"),
+            ApiResponse(responseCode = "500", description = "Intern serverfeil")
+        ]
+    )
+    @Unprotected
+    @GetMapping("/grunnlagsdata")
+    fun hentKalkuleringsinformasjon(): GrunnlagsDataDto {
```

Comment:
Slenger du på noen cache headere på denne eller? Er jo ikke vits å spørre serveren mer enn en gang om dagen for denne statiske dataen.